content/questions/actions/question-119.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "When can you delete workflow runs (Select two.)"
+archetype: "questions"
+title: "Question 119"
+draft: false
+---
+
+> https://docs.github.com/en/actions/managing-workflow-runs/deleting-a-workflow-run
+- [x] When workflow run has been completed
+- [x] When workflow run is two weeks old
+- [ ] When workflow run is 90 days old
```

Comment:
You cannot delete workflow runs that are 90 days old? Can you provide the source for that

Reply:
my bad, I mixed it with artifact (maximum 90 days for artifacts in public repositories and 400 days in private repositories). I have changed it to 10 day